app/models/manageiq/providers/kubevirt/inventory/parser/partial_refresh.rb

```diff
@@ -76,16 +69,12 @@ def parse
   private
 
   def get_object_ids(notices)
-    ids = notices.map { |notice| notice.object.metadata.uid }
+    ids = notices.map { |notice| notice.metadata.uid }
```

Comment:
I think we can replace these three lines with one:
notices.map { |notice| notice.metadata.uid }.uniq

Reply:
Done